frontend/package.json

```diff
@@ -24,11 +24,13 @@
     "dependencies": {
         "@ant-design/icons": "^4.7",
         "@chakra-ui/icons": "^2.1",
+        "@chakra-ui/react": "^2.8.1",
         "@chakra-ui/react-use-disclosure": "^2.1",
         "@heroicons/react": "^1.0.6",
         "@monaco-editor/react": "^4.6",
         "@primer/octicons-react": "^17",
-        "@redpanda-data/ui": "^3.30.1",
+        "@redpanda-data/ui": "file:.yalc/@redpanda-data/ui",
```

Comment:
let's revert the change to `yalc`, we can use a beta release, or focus on the datatable updates in UI library 1st?

Reply:
Yeas, please let's just temporarily keep these here until a BETA package is released